Cargo.toml

```diff
@@ -27,8 +27,8 @@ basic-cookies   = { version = "^0.1", optional = true }
 bytes           = "1"
 chrono          = { version = "^0.4", features = ["serde"] }
 clap            = "^2.33"
-cryptoki        = { version = "^0.3", optional = true }
-cryptoki-sys    = { version = "=0.1.4", optional = true } # pin cryptoki-sys because of compilation issues on various systems
+cryptoki        = { version = "0.7", optional = true}
+#cryptoki-sys    = { version = "=0.1.4", optional = true } # pin cryptoki-sys because of compilation issues on various systems
```

Comment:
Why not delete this line rather than comment it out?

Reply:
That is a very good question!